src/unlimited.cpp

```diff
@@ -424,6 +424,19 @@ void UnlimitedSetup(void)
     excessiveAcceptDepth = GetArg("-excessiveacceptdepth", excessiveAcceptDepth);
     LoadTweaks(); // The above options are deprecated so the same parameter defined as a tweak will override them
 
+    // If the user configures it to 1, assume this means default
+    if (miningForkTime.Value() == 1)
+        miningForkTime = Params().GetConsensus().nov2018ActivationTime;
+    if (miningSvForkTime.Value() == 1)
+        miningSvForkTime = Params().GetConsensus().nov2018ActivationTime;
+
+    if (miningForkTime.Value() != 0 && miningSvForkTime.Value() != 0)
+    {
```

Comment:
does this cover also the case where those values are changed at runtime via `bitcoin-cli`?

Guess we should enforce it via an ad-hoc tweaks validator.

Reply:
good point. thx!